update-script/rebase-official.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+# Due to how much time has passed since the preflight steps and when the PR is created trying to merge local changes to master can cause merge conflicts. 
+# This script is for quickly rebasing in case of merge conflicts when trying to push to master. 
+
+# Set initial variables
+git_directory="/home/ec2-user/official-images-output/official-images"
```

Comment:
maybe this should be a script argument as I think hard-coding this adds additional step for technician to create this directory and put contents inside it.

Reply:
Could this be $HOME instead of /home/ec2-user to make it more resistant to eg. running as a different user?

Reply:
@mysteriouspants yup you're right let me update that.